### ISSUE TITLE
Make patches targeting new released 10.0.0-27 NATIVE_FIRM

### ISSUE THREAD

**Issue by soarqin** [roxas75/rxTools#240]
Nintendo may remove 9.6.0-24 NATIVE_FIRM from their cdn sooner or later like they did to other old firmware titles, so we had better start to make patches for newest NATIVE_FIRM.
As researches from 3dbrew, there is no change to patched parts, there are just changes in address offsets, I think it is not hard to do it.

**Comment by nastys** [comment]
Has this been fixed yet?

**Comment by soarqin** [comment]
nope, I do not know how to dump 10.0.0+ firm memory for patches analyze

**Comment by nastys** [comment]
OK, I don't know how to do it either, so I'll label this issue as enhancement.
